web/src/components/molecules/Common/MultiValueField/index.tsx

```diff
@@ -0,0 +1,91 @@
+import styled from "@emotion/styled";
+import React, { ChangeEvent } from "react";
+
+import Button from "@reearth-cms/components/atoms/Button";
+import Icon from "@reearth-cms/components/atoms/Icon";
+import { InputProps } from "@reearth-cms/components/atoms/Input";
+import { TextAreaProps } from "@reearth-cms/components/atoms/TextArea";
+import { useT } from "@reearth-cms/i18n";
+
+import { moveItemInArray } from "./moveItemArray";
+
+type Props = {
+  className?: string;
+  value?: (string | number)[];
+  onChange?: (value: (string | number)[]) => void;
+  FieldInput: React.FunctionComponent<any>;
+} & TextAreaProps &
+  InputProps;
+
+const MultiValueField: React.FC<Props> = ({
+  className,
+  value = [],
+  onChange,
+  FieldInput,
+  ...props
+}) => {
+  const t = useT();
+  const handleInput = (e: ChangeEvent<HTMLInputElement>, id: number) => {
+    onChange?.(
+      value?.map((valueItem, index) =>
+        index === id ? (typeof e === "number" ? e : e.target.value) : valueItem,
+      ),
+    );
+  };
+
+  const deleteInput = (key: number) => {
+    onChange?.(
+      value.filter((_, index) => {
+        return index !== key;
+      }),
+    );
+  };
```

Comment:
Same as other components. Should be useCallback and deleteInput should be handleInputDelete